src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -468,7 +415,35 @@ def get_job_logs(
                 "lines": logs,
             }
         except Exception as e:
-            raise transform_job_exception(e, "Unable to retrieve job logs")
+            return {
+                "job_id": job.job_id,
+                "batch_id": job.batch_id,
+                "error": e.message,
+            }
```

Comment:
Instead of throwing an error, which the frontend laboriously converts into a job logs message, return the error message as part of the job logs results.

Frontend changes will be coming in the next PR.